src/views/tables/PositionsTable.tsx

```diff
@@ -554,22 +480,25 @@ export const PositionsTable = forwardRef(
     const { isSlTpLimitOrdersEnabled } = useEnvFeatures();
     const { isTablet } = useBreakpoints();
 
+    // todo this uses the old subaccount id for now
     const isAccountViewOnly = useAppSelector(calculateIsAccountViewOnly);
-    const perpetualMarkets = orEmptyRecord(useAppSelector(getPerpetualMarkets, shallowEqual));
-    const assets = orEmptyRecord(useAppSelector(getAssets, shallowEqual));
 
-    const openPositions = useAppSelector(getExistingOpenPositions, shallowEqual) ?? EMPTY_ARR;
+    const perpetualMarkets = orEmptyRecord(useAppSelector(selectRawMarketsData));
```

Comment:
why use rawMarketsData instead of `selectAllMarketsInfo`?

Reply:
w/ rebase you'll be able to get assetId, tickSizeDecimals, etc.

Reply:
it might make sense to convert all values to bigNumber within calculateMarkets as well